.gitignore

```diff
@@ -39,3 +39,8 @@ yarn-error.log*
 junit.xml
 
 *.tsbuildinfo
+
+/.pnp.js
+/.yarn/cache/
+/.yarn/unplugged/
+/.yarn/build-state.yml
```

Comment:
If you don't plan to use zero-installs you could add `enableGlobalCache: false` to the `.yarnrc.yml`, so that yarn would use global cache directly instead of copying it to the `.yarn/cache` during install

Reply:
yeah, I dunno what approach we wanna take, for now I just ignored as we haven't decided anything. can use the conflig flag rather than gitignore though, thanks!